crates/rover-client/src/query/config/whoami.graphql

```diff
@@ -0,0 +1,10 @@
+query WhoAmIQuery {
+  me {
+    __typename
+    name
```

Comment:
I think I would not recommend reading this field and displaying it to the user as "name". This isn't well documented (our schema isn't too documented, sorry) but this interface field has two real implementations: Service.name and User.name.

Service.name is actually a deprecated field: the value in it is referred to as "title" these days in our UI and has a non-deprecated Service.title field. (We thought it was confusing to have the name and ID be distinct things — people generally referred to the the ID as the name. So we changed it so the things are "title" and "ID", and often use name to mean the latter. So Rover shouldn't use name to mean the former.)

User.name is literally just User.id without the first few characters (`gh.` etc).  I would recommend not querying `Identity.field` or displaying a `name:` line, but instead doing a `... on Service { title }` and showing that field as "title".